demos/common/cpp/models/src/style_transfer_model.cpp

```diff
@@ -0,0 +1,101 @@
+/*
+// Copyright (C) 2021 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+*/
+
+#include "models/style_transfer_model.h"
+
+#include "utils/ocv_common.hpp"
+#include <utils/slog.hpp>
+
+#include <string>
+#include <vector>
+#include <memory>
+
+using namespace InferenceEngine;
+
+StyleTransferModel::StyleTransferModel(const std::string& modelFileName) :
+    ImageModel(modelFileName, false) {
+}
+
+template<typename T>
+void tell_me_type(T t, std::string name) {
+    std::cout << name << " " << __PRETTY_FUNCTION__ << std::endl;
+}
```

Comment:
Where you use this function? If it don't used, please delete it.

Reply:
Done